lib/filewatch/discoverer.rb

```diff
@@ -36,6 +36,13 @@ def discover
     private
 
     def can_exclude?(watched_file, new_discovery)
+      if watched_file.file_ignorable?
```

Comment:
If file should be ignored - we should not include it in list of watched files at all